hermeto/interface/cli.py

```diff
@@ -210,8 +210,11 @@ def _looks_like_json(value: str) -> bool:
 
```

Comment:
nitpick:

> Fixes commit https://github.com/hermetoproject/hermeto/commit/01140a3f034dabd5e97fac631a8f8ceaf0bd416d.

No periods after commit digests please, it renders incorrectly in git history (unless viewed in GH UI) :) .
```
Fixes: <digest>
```
is perfectly sufficient.